packages/main/src/plugin/tasks/progress-impl.ts

```diff
@@ -78,14 +83,34 @@ export class ProgressImpl {
     );
   }
 
+  protected getTaskAction(options: extensionApi.ProgressOptions): TaskAction | undefined {
+    if (!options.details) return undefined;
+
+    if (!this.navigationManager.hasRoute(options.details.routeId)) {
+      console.warn(`cannot created task action for unknown routeId ${options.details.routeId}`);
+      return undefined;
+    }
+
+    return {
+      name: 'View',
```

Comment:
I still believe `View details` or `Inspect` is more accurate but as it's not part of the spec, it can be changed over time (or not) later.

can merge and discuss it during UX discussions